build_9.0/wb3.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
We will probably need a new directory in `data/world-builder/` or `data/initial-temperature/world-builder` for these files.
Also: Is it possible to add comments to these files?

Reply:
>We will probably need a new directory in `data/world-builder/` or `data/initial-temperature/world-builder` for these files.

I think it will be better to put it in the end into a cookbook, since it is not really data, just an example input.

>Also: Is it possible to add comments to these files?

Unfortunately for us, they removed support for comments from JSON because people where apparently abusing it as macro's, and some other reasons (see https://groups.yahoo.com/neo/groups/json/conversations/topics/156 ).  But what can be done is just adding a field `"comment":"and put the comment here",`. I could them exempt the comment field from needing declaration, or ignore the field when reading stuff in. 

An other thing I could manually parse the file in advance and remove comment lines, but then we would diverge from the JSON standard, making implementation into other codes slightly more difficult.

Reply:
Yes, manually parsing is not an option and extending the file format with comments is also not an option. I do like the idea of a "comment" field.

Reply:
Ok, that is good enough for me. I just want to have the option to put in stuff like "this is model X with data taken from publication Y" or something of that sort.